infrastructure/development/apim.tf

```diff
@@ -0,0 +1,102 @@
+# API management
+
+## Create and configure the API management service
+
+resource "null_resource" "azurerm_apim" {
+  triggers = {
+    azurerm_function_app_id     = "${azurerm_function_app.azurerm_function_app.id}"
+    azurerm_resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
+    provisioner_version         = "1"
+  }
+
+  provisioner "local-exec" {
+    command = "${join(" ", list(
+      "ts-node ${var.apim_provisioner}",
```

Comment:
APIM support has been added to the azure provider, consider moving the apim creation to terraform and keeping just the configuration in the provisioner: https://www.terraform.io/docs/providers/azurerm/r/api_management.html